src/coreclr/nativeaot/Runtime/CommonMacros.h

```diff
@@ -293,14 +293,11 @@ typedef uint8_t CODE_LOCATION;
 
 typedef bool CLR_BOOL;
```

Comment:
Delete?

Reply:
Looks like it's still used as the return value for `HandleTableRestrictedCallbackFunction`.

Reply:
And TryGetTaggedMemoryCallback.

These correspond to the following managed methods:
- https://github.com/dotnet/runtime/blob/37284fce7ea631d1d161645b465a4e7d2570a49e/src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.NativeAot.cs#L428-L434
 - https://github.com/dotnet/runtime/blob/b1968e7aa8d56a088e8be6817d5240fb345f901c/src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ObjectiveCMarshal.NativeAot.cs#L58-L59

`CLR_BOOL` is not quite right for neither of these, but I do not think that this mismatch is going to cause any real problem. It would be nice to clean this up in a follow up.